jsonnet/rules.jsonnet

```diff
@@ -96,6 +96,27 @@
           },
         ],
       },
+      {
+        name: 'openshift-ingress.rules',
+        rules: [
+          {
+            expr: 'sum by (code) (rate(haproxy_server_http_responses_total[5m]) > 0)',
+            record: 'code:cluster:ingress_http_request_count:rate:sum',
+          },
+          {
+            expr: 'sum by (frontend) (rate(haproxy_frontend_bytes_in_total[5m]))',
+            record: 'frontend:cluster:ingress_frontend_bytes_in:rate:sum',
+          },
+          {
+            expr: 'sum by (frontend) (rate(haproxy_frontend_bytes_out_total[5m]))',
+            record: 'frontend:cluster:ingress_frontend_bytes_out:rate:sum',
```

Comment:
each of these should be `...rate5m...`, otherwise this looks good to me

Reply:
fixed